modules/manage/pages/security/iam-roles.adoc

```diff
@@ -117,6 +117,15 @@ If you are using Google Cloud Platform as your cloud provider, you must satisfy
 
 NOTE: A full access policy with all storage bucket permissions is required for Tiered Storage.
 
+=== Azure prerequisites
+
+If you are using Microsoft Azure as your cloud provider, you must satisfy the following prerequisites:
+
+. https://learn.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal#create-a-storage-account-1[Create an Azure storage account]^.
+. https://learn.microsoft.com/en-us/azure/storage/blobs/blob-containers-portal#create-a-container[Create a container] in the storage account.
+. https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/how-manage-user-assigned-managed-identities?pivots=identity-mi-methods-azp#create-a-user-assigned-managed-identity[Create a user-assigned managed identity] with the appropriate permissions.
```

Comment:
@andijcr @c4milo @deniscoady Does anyone have details on what the specific permissions are required on the managed identity?

Reply:
should be `Storage Blob Data Contributor`

Reply:
I'm not sure if there is a different process; for my specific tests, I created a user-assigned managed identity, and then in the roles section of the storage account, I added it with the `Storage Blob Data Contributor` permission. 
so the wording `[Create a user-assigned managed identity] with the appropriate permissions.` might not be totally correct